Cookbook/Technical-Documents/TechnicalDebt.md

```diff
@@ -7,62 +7,148 @@ Technical debt and legacy code has a sad tendency to build up in any software pr
+| NHSRegistrationStep2ViewController | NRX-186 | | Used when `isV4NHSRegistrationStep1And2Enabled == false` |
+| RedemptionViewModel | CNSMR-921 | | Used when `isNewRedemptionEnabled ==false ` |
+| OnboardingViewModel | AV-118 | | Used when `isNewWelcomeScreenEnabled == false` |
+
+## Bento View Controllers that need to be Updated to use the Design Library.
+
+`Bento` and its component library `BentoKit` predates the design library. Consequenly there are a number of `BoxRenderer` classes that need to be refactored to employ the design library.
+
+### Renderers that do not employ the design library
+
+| Renderer | JIRA tickets | Code Ownership | Comments |
+| ---------- | -------- | -------- |  -------- |
+| PrescriptionRenderer | | | |
+| PrescriptionDeliveryOptionsRenderer | | | |
+| ChatMenuRenderer | | | |
+| SymptomSelectorRenderer | | | |
+| MessageReportRenderer | | | |
+| PGMConditionDetailsRenderer | | | |
+| PGMConditionDetailsRenderer | | | |
+| ClinicalRecordsRenderer | | | |
+| PGMReportRenderer | | | |
+| GPAtHandDetailsRenderer | | | |
+| PharmaciesRenderer | | | |
+| ClinicalRecordsRenderer | | | |
+| MetricDetailsRenderer | | | |
+| QuestionnaireRenderer | | | |
+| UserFeedbackRatingRenderer | | | |
+| MapSearchResultsRenderer | | | |
+| PlaceListRenderer | | | |
+| ChooseAssessmentRenderer | | | |
+| DetailRenderer | | | |
+| HealthOverviewRenderer | | | |
+| RiskSortMenuRenderer | | | |
+| DiseaseRisksRenderer | | | |
+| ActionsRenderer | | | |
+| AdditionalInfoFormRenderer | | | |
+| RatingRenderer | | | |
+| ChooseStateRenderer | | | |
+| AddressListRenderer | | | |
+| ShopConfirmationRenderer | | | |
+| ChooseProviderRenderer | | | |
+| PaymentPlansRenderer | | | |
+| SubscriptionConfirmationRenderer | | | |
+| SubscriptionsRenderer | | | |
+| EligibilityCheckRenderer | | | |
+| NHSSwitchV2Renderer | | | |
+| NHSConfirmationSwitchRenderer | | | |
+| NHSSummaryRenderer | | | |
+| NotificationsV2Renderer | | | |
+| PaymentRenderer | | | Used when `isNewPaymentsAvailable == false` |
+| AddCardRenderer | | | |
+| ChoosePaymentMethodRenderer | | | |
+| TestsAndKitsKitsInfoRenderer | | | |
+| ChooseSponsorRenderer | | | |
+
+### Helpers that are not using the design library
+
+| Renderer | JIRA tickets | Code Ownership | Comments |
+| ---------- | -------- | -------- |  -------- |
+| InfoRendererV2 | | | |
+| RichTextRenderer | | | Uses both the design library and `BentoKit` |
+| GetStartedRenderer | | | Uses both the design library and `BentoKit` |
+| IntroductionRenderer  | | | Uses both the design library and `BentoKit` |
+
+### Renderers that mixes the design library with BentoKit
+
+Direct references to `BentoKit` components are probably left overs from porting the renderer to use the design library.
+
+| Renderer | JIRA tickets | Code Ownership | Comments |
+| ---------- | -------- | -------- |  -------- |
+| ManageSyncRenderer | | | |
+| DashboardRenderer | | | |
+| NHSOnboardingV2Renderer | | | |
```

Comment:
```suggestion
| NHSOnboardingV2Renderer | | | |
| NHSSummaryRenderer | | | |
| PDSRetryRenderer | | | |
| ContactSupportRenderer | | | |
```

Reply:
@diegopetrucci `NHSOnboardingV2Renderer` initialises `Component.TitledDescription` directly in a couple of places.